<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Taomkcap2</class>
 <widget class="QDialog" name="Taomkcap2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/backgroup/picture/bg9.png);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 54, 58);</string>
    </property>
    <property name="text">
     <string>Secondary password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 54, 58);</string>
    </property>
    <property name="text">
     <string>Please enter your secondary password to </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 54, 58);</string>
    </property>
    <property name="text">
     <string>confirm the transaction.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
border-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
color: rgb(0, 0, 0);
font: 600 14pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
border-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
color: rgb(0, 0, 0);
font: 600 14pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
border-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
color: rgb(0, 0, 0);
font: 600 14pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
border-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
color: rgb(0, 0, 0);
font: 600 14pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
border-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
color: rgb(0, 0, 0);
font: 600 14pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/nenchu1/picture/Dark_Purple.webp);
border-image: url(:/nenchu1/picture/Dark_Purple.webp);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>confirm</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
border-image: url(:/nenchu1/picture/y-nghia-mau-trang-1.jpg);
color: rgb(0, 0, 0);
font: 600 14pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
